src/PowerShell/Microsoft.WinGet.Client/Module/Microsoft.WinGet.Client.nuspec

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="utf-8"?>
+<package >
+  <metadata>
+    <id>Microsoft.WinGet.Client</id>
+    <version>0.1.0</version>

```

Comment:
`$version$`

This would actually be better a passed in value since every release, the value changes